src/components/DropdownMenu.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { DropdownButton, Dropdown } from "react-bootstrap";
+
+const DropdownMenu = ({ options, selectedColor, setSelectedChangeColor }) => {
+  const renderedOptions = options.map((option) => {
```

Comment:
to co nap[isales jest spoko, ale co ty na to, żeby
how about using option.filter before map? This is liek a .map method, but allows to skip elements that don't spelniaja condition:
```
option
          .filter((option) => option.value !== selectedColor.value) // to zwroci nowa liste optionsow (bez równyhkolorowi)
          .map((option) =>  <Dropdown.Item //tutaj to co masz dalej
                                                   key={option.value}
                                                  onClick={() => setSelectedChangeColor(option)}
                                                   >
                                                     {option.label}
                                                </Dropdown.Item>)
```